src/main/java/com/cloudbees/plugins/deployer/DeployEvent.java

```diff
@@ -36,6 +37,7 @@
 /**
  * @author stephenc
  */
+@SuppressFBWarnings(value = "SE_BAD_FIELD")
```

Comment:
Why? This certainly looks like a bug to me. `AbstractBuild` cannot be serialized. So either make the field `transient`, or remove `implements Serializable` if no one is actually serializing it, or explain why this code is not doing what it looks like.